base/redhat-8/install.sh

```diff
@@ -26,14 +26,14 @@ microdnf -y --nodocs install glibc-langpack-en
 #We get around the gen above by forcing the language install, and then point to it.
 export LANG=en_US.utf8
 
-microdnf -y --nodocs install wget sudo shadow-utils procps
+microdnf -y --nodocs install wget sudo shadow-utils procps tar
 #install busybox direct from the multiarch since epel isn't availible yet for redhat8
 wget -O /bin/busybox https://busybox.net/downloads/binaries/1.28.1-defconfig-multiarch/busybox-`arch`
 chmod +x /bin/busybox
-microdnf -y --nodocs install gcc redhat-rpm-config python2-devel libffi-devel openssl-devel tar
+microdnf -y --nodocs install python2-pip python2-devel redhat-rpm-config gcc libffi-devel openssl-devel
 pip2 --no-cache-dir install requests ansible
-microdnf -y remove gcc libffi-devel openssl-devel
-microdnf clean all
```

Comment:
Why remove the `microdnf clean all`?

Reply:
There's another `microdnf clean all` on line 59 - I guess it doesn't hurt to have both, but I figured it was an unnecessary line.